livekit-agents/livekit/agents/voice_assistant/transcription_manager.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import time
+from typing import List
+
+from livekit import rtc
+
+
+class TranscriptionManager:
+    def __init__(self, room: rtc.Room, participant_identity: str):
+        self._participant_identity = participant_identity
+        self._segments: List[rtc.TranscriptionSegment] = []
+        self._track_sid = ""
+        self._room = room
+        self._current_id = 1
```

Comment:
this should be an uuid, we don't want it to conflict between TranscriptionManager instances.